### ISSUE TITLE
Missing form label

### ISSUE THREAD

**Issue by cynthiaguilas** [Virtualahan/virtualahan-com#30]
A form control does not have a corresponding label.

If a form control does not have a properly associated text label, the function or purpose of that form control may not be presented to screen reader users. Form labels also provide visible descriptions and larger clickable targets for form controls.

If a text label for a form control is visible, use the <label> element to associate it with its respective form control. If there is no visible label, either provide an associated label, add a descriptive title attribute to the form control, or reference the label(s) using aria-labelledby. Labels are not required for image, submit, reset, button, or hidden form controls.

![Footer - missing form label](https://user-images.githubusercontent.com/81419695/113472907-b22a7980-9498-11eb-97ae-623945e050b4.JPG)
